src/bindgen/bindings.rs

```diff
@@ -209,9 +212,30 @@ impl Bindings {
             return;
         }
 
+        if self.config.package_version {
+            out.new_line_if_not_start();
+            match self.config.language {
+                Language::C | Language::Cxx => {
+                    write!(out, "/* Package version: {} */", self.package_version,);
+                }
+                Language::Cython => {
+                    write!(out, "''' Package version: {} '''", self.package_version,);
+                }
+            }
+
+            out.new_line();
+        }
+
         if let Some(ref f) = self.config.header {
```

Comment:
Same here, this should probably remain as it was.

Reply:
So with these two commits, I removed the user input modification - and adjusted also the package_version test. Yes I see, it could be dangerous to change input.